chunks/lang-node/Dockerfile

```diff
@@ -20,3 +20,5 @@ RUN curl -fsSL https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh |
     && echo ". ~/.nvm/nvm-lazy.sh"  >> /home/gitpod/.bashrc.d/50-node
 # above, we are adding the lazy nvm init to .bashrc, because one is executed on interactive shells, the other for non-interactive shells (e.g. plugin-host)
 COPY --chown=gitpod:gitpod nvm-lazy.sh /home/gitpod/.nvm/nvm-lazy.sh
+
+RUN npm config set update-notifier false
```

Comment:
@gengjiawen this feels like an individual preference. Could I ask you to add to a custom image of yours, or, a related project .gitpod.yml as a task? 

Alternatively, there are some additional options [here](https://github.com/yeoman/update-notifier#user-settings).

Reply:
Actually I want it to be default, since it bring to much noise.